### PR TITLE
Remove appinfo

### PR DESCRIPTION
**Pull request by naslundx**
This doesn't belong in this project.

### CODE REVIEW

File: webtraversallibrary/scraper.py [hunk 1]
```diff
@@ -221,7 +220,6 @@ def _create_snapshot(self) -> PageSnapshot:
             "full_page_size": (self.config.browser.width, self.js.get_full_height()),
             "device_pixel_ratio": get_device_pixel_ratio(self.driver),
             "num_elements": num_elements,
-            "commit_hash": get_commit_hash(),
```
Comment (vonNiklasson) [inline webtraversallibrary/scraper.py:224]: What was this used for before? Would it be reasonable to include the package version instead to make it standalone from git?

Comment (naslundx) [reply webtraversallibrary/scraper.py:224]: That's not a bad idea. This was originally used in a certain klarna-related project starting with D and ending with W, but relies on a very custom `.gitinfo` to be generated and added. It'd make more sense for that project's code to add this info to snapshots (trivial to do) if you actually need it (probably not).

Comment (naslundx) [reply webtraversallibrary/scraper.py:224]: Updated

**Review by vonNiklasson — APPROVED** [review]
LGTM!

**Pull request merged**